VirtoCommerce.Platform.Web/Scripts/app/navigation/blade/bladeNavigation.js

```diff
@@ -392,10 +399,12 @@ angular.module('platformWebApp')
         setError: function (msg, blade) {
             if (blade) {
                 blade.isLoading = false;
-                blade.error = msg;
+                blade.error = $.trim(msg);
+                if (blade.error.length > 55)
```

Comment:
Such a logic can be performed using css styles, the controller logic should ignore the details of the presentation